### ISSUE TITLE
[Rename] attribute rename doesn't work

### ISSUE THREAD

**Issue by bleis-tift** [fsprojects/VisualFSharpPowerTools#797]
```fsharp
type SampleAttribute () =
  inherit System.Attribute()

[<Sample>]
type Class () = class end
```

rename `SampleAttribute` to `SimpleAttribute`:

```fsharp
type SimpleAttribute () =
  inherit System.Attribute()

[<Sample>] // not renamed
type Class () = class end
```

**Comment by vasily-kirichenko** [comment]
Before
```fsharp
type SampleAttribute () =
  inherit System.Attribute()

[<Sample>]
type Class () = class end

let _ = SampleAttribute()
```
1. Cursor at attribute definition (`SampleAttribute` -> `Sample1Attribute`)
```fsharp
type Sample1Attribute () =
  inherit System.Attribute()

[<Sample1Attribute>]  // wrongly added "Attribute" suffix
type Class () = class end

let _ = Sample1Attribute()
```
2. Cursor at attribute applying (`Sample` -> `Sample1`)
```fsharp
type Sample1 () = // the suffix is removed
  inherit System.Attribute()

[<Sample1>]
type Class () = class end

let _ = Sample1() // the suffix is removed
```
3. Cursor at attribute creation, like a normal class (`SampleAttribute` -> `Sample1Attribute`)
```fsharp

type Sample1Attribute () =
  inherit System.Attribute()

[<Sample1Attribute>] // the suffix added
type Class () = class end

let _ = Sample1Attribute()
```

**Comment by dungpa** [comment]
Ok, thanks.

I'm on vacation now. I will take a look again when I'm back after a few days.

**Comment by dungpa** [comment]
@vasily-kirichenko I fixed case 2 at 529a959de183a9ba9f0dadca6621111a75294787.

Case 1 (and 3) is still syntactically correct. I think it's a minor issue but requires nasty changes to fix. Users can preserve the original short forms by renaming at attribute usages.
